pkgs/development/python-modules/disposable-email-domains/default.nix

```diff
@@ -0,0 +1,44 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  setuptools,
+  check-manifest,
+  pytestCheckHook,
+}:
+
+buildPythonPackage rec {
+  pname = "disposable-email-domains";
+  version = "0.0.126";
+  pyproject = true;
+
+  # No tags on GitHub
+  src = fetchPypi {
+    pname = "disposable_email_domains";
+    inherit version;
+    hash = "sha256-fPxAOOT7yIT7DoupnUlC6iQQWBMw5iaNoH8hjjWBKMI=";
+  };
+
+  build-system = [
+    setuptools
+  ];
+
+  optional-dependencies = {
+    dev = [
+      check-manifest
```

Comment:
What would a Nixpkgs user use this for? Usually optional-dependencies like `dev` or `lint` is not included in pythonPackages.